### PR TITLE
Fix for yaml parsing issue

### PR DESCRIPTION
**Pull request by Divya063**
#### What type of PR is this:

/kind bug

#### What this PR does / why we need it:
Getting error: ` decoding note map: yaml: line 3: found character that cannot start any token`  while generating release notes for v1.25.0-alpha.3

### CODE REVIEW

File: releases/release-1.25/release-notes/maps/pr-109251-map.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 pr: 109251
-releasenote:
-  text: `MaxUnavailable` for `StatefulSets`, allows faster `RollingUpdate` by taking down
-    more than 1 pod at a time. The number of pods you want to take down during a `RollingUpdate`
-    is configurable using `maxUnavailable` parameter.
```
Comment (saschagrunert) [inline releases/release-1.25/release-notes/maps/pr-109251-map.yaml:5]: I think this has been generated from the krel input, right @csantanapr? If so, then we may have a bug here.

Comment (csantanapr) [reply releases/release-1.25/release-notes/maps/pr-109251-map.yaml:5]: @saschagrunert, this text was modified during PR review process by a person (ie me)

Comment (csantanapr) [reply releases/release-1.25/release-notes/maps/pr-109251-map.yaml:5]: I don't see what the error `decoding note map: yaml: line 3: found character that cannot start any token` is referring to

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**